src/frontend/components/Felleskomponenter/PeriodeOppsummering/PeriodeOppsummering.tsx

```diff
@@ -32,12 +34,14 @@ const PeriodeOppsummering: React.FC<{
     nummer: number;
     fjernPeriodeCallback?: () => void;
     fjernKnappSpråkId?: string;
-    tittelSpråkId: string;
+    fjernKnappTekst?: LocaleRecordBlock;
+    tittelSpråkId?: string;
```

Comment:
Er det meningen at dette blir gjort optional i denne PR'en? 🤔 ser ut som at det i såfall ikke blir tatt høyde for at den kan være optional lenger ned her